src/silicone/multiple_infillers/infill_all_required_emissions_for_openscm.py

```diff
@@ -150,8 +161,11 @@ def infill_all_required_variables(
         database = database.filter(year=output_timesteps)
         to_fill = to_fill.filter(year=output_timesteps)
     else:
-        database = database.filter(time=output_timesteps)
-        to_fill = to_fill.filter(time=output_timesteps)
+        # TODO: this filter switchup is needed because of a problem with filter.
+        #  If the pyam bug clears we can remove it.
```

Comment:
Is there an issue in pyam for this?

Reply:
No, I was sketching out a minimal-case bug report before the break. It's there now, https://github.com/IAMconsortium/pyam/issues/367

Reply:
beauty